test/lib/extractor/extractor.spec.ts

```diff
@@ -80,6 +80,27 @@ describe("extractImageContent", () => {
     });
   });
 
+  describe("Kaniko Image Archives (expected to fall back to DockerExtractor)", () => {
```

Comment:
What do you mean by "expected to fall back to DockerExtractor"?

Reply:
cleaning this up, from the test case copy above